istioctl/pkg/ztunnelconfig/ztunnelconfig.go

```diff
@@ -42,50 +48,119 @@ const (
 	defaultProxyAdminPort = 15000
 )
 
-var (
-	workloadsNamespace string
-	verboseProxyConfig bool
-
-	address, node string
-
-	// output format (json, yaml or short)
-	outputFormat string
-
-	proxyAdminPort int
-
-	configDumpFile string
-
-	labelSelector = ""
-)
-
 func ZtunnelConfig(ctx cli.Context) *cobra.Command {
 	configCmd := &cobra.Command{
 		Use:   "ztunnel-config",
 		Short: "Update or retrieve current Ztunnel configuration.",
 		Long:  "A group of commands used to update or retrieve Ztunnel configuration from a Ztunnel instance.",
-		Example: `  # Retrieve summary about workload configuration for a given Ztunnel instance.
-	istioctl ztunnel-config <workload> <ztunnel-name[.namespace]>`,
+		Example: `  # Retrieve summary about workload configuration
+  istioctl ztunnel-config workload
+
+  # Retrieve summary about certificates
+  istioctl ztunnel-config certificates`,
 		Aliases: []string{"zc"},
 	}
 
-	configCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", summaryOutput, "Output format: one of json|yaml|short")
-	configCmd.PersistentFlags().IntVar(&proxyAdminPort, "proxy-admin-port", defaultProxyAdminPort, "Ztunnel proxy admin port")
-
 	configCmd.AddCommand(logCmd(ctx))
 	configCmd.AddCommand(workloadConfigCmd(ctx))
+	configCmd.AddCommand(certificatesConfigCmd(ctx))
+	configCmd.AddCommand(servicesCmd(ctx))
 
 	return configCmd
 }
 
+type Command struct {
+	Name string
+}
+
+func certificatesConfigCmd(ctx cli.Context) *cobra.Command {
+	common := new(commonFlags)
+	cmd := &cobra.Command{
+		Use:   "certificate",
+		Short: "Retrieves certificate for the specified Ztunnel pod.",
+		Long:  `Retrieve information about certificates for the Ztunnel instance.`,
+		Example: `  # Retrieve summary about workload configuration for a randomly chosen ztunnel.
```

Comment:
This is a little weird? Ideally random is fine I guess since they _should_ have all the same state, but I would rather just require users to specify (ztname || nodename) and error out if none provided, randomly picking a target seems weird - if you aren't being specific, why run this command?

Reply:
I guess for certs it's less useful. For everything else it should be identical so it seemed useful. 

I don't mind removing it, it's easy to simulate by passing `ds/ztunnel` directly anyways